### DISCUSSION TITLE
Folder browsing

### DISCUSSION THREAD

**Discussion by CoolHandDevil505** [BazzaCuda/CustomMenu#22] [General]
Hi,
Wiki says: "You can even browse folders and, if you wish, subfolders, direct from your desktop, with options to limit what types of files are displayed in the submenus.
Alternatively, you can include subfolders and CustomMenu will become a live disk-browsing facility, with a submenu for every subfolder it finds"

But how can it be done? Maybe it is obvious, but I did not find it 🤔

**Comment by BazzaCuda** [comment]
Hi, the explanation is here: https://github.com/BazzaCuda/CustomMenu/wiki/Browsing-files-and-folders

Any questions, let me know.

**Reply by CoolHandDevil505** [reply]
Sorry, did not see this page…
Thank you!

**Reply by BazzaCuda** [reply]
No probs 👍
